content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -410,6 +410,8 @@ A PodCertificateRequest has the following spec fields:
 * `pkixPublicKey`: The public key for which the certificate should be issued.
 * `proofOfPossession`: A signature demonstrating that the requester controls the
   private key corresponding to `pkixPublicKey`.
+* `unverifiedUserAnnotations`: A map that allows the user to pass additional
+  information to the signer implementation.
```

Comment:
Mention that unlike the rest of the fields, the API server does not validate these values in any way and that signer implementations should be very careful when consuming this data.  Signers must not inherently trust this data without first performing the appropriate verification steps.

Reply:
Duplicate the information in both places.  It is better to be verbose than to have CVEs in the ecosystem.